integration_tests/README.md

```diff
@@ -303,3 +303,66 @@ The marks you care about are all in marks.py
 
 For the most part you can ignore this file. It provides the underlying Spark session to operations that need it, but most tests should interact with
 it through `asserts.py`.
+
+## Guidelines for Testing
+
+When support for a new operator is added to `rapids-plugin-4-spark`, it is recommended that
+the following conditions be covered in its corresponding integration tests:
+
+### 1. Cover all supported data types
+Ensure that tests cover all data types supported by the added operation. An exhaustive list of data types supported in 
+Spark is available [here](https://spark.apache.org/docs/latest/sql-ref-datatypes.html). These include:
+   * Numeric Types (`ByteType`, `ShortType`, `IntegerType`, etc.)
```

Comment:
it would be good to call out decimal on its own. I am honestly not an expert at decimals, but it would be good to figure out if there are specific gotchas with it.

Reply:
Can we just list out all the types supported by us instead of referring to the Spark doc? That will make this a self-contained doc and we can add types as we support more

Reply:
Listed out all types, instead of `etc.`.